Windows 10 import error

Hey, It seems like the same issue as #38 prevails even in 1.7.1:
```
    import faiss
C:\Python\lib\site-packages\faiss\__init__.py:18: in <module>
    from .loader import *
C:\Python\lib\site-packages\faiss\loader.py:65: in <module>
    from .swigfaiss import *
C:\Python\lib\site-packages\faiss\swigfaiss.py:13: in <module>
    from . import _swigfaiss
E   ImportError: DLL load failed while importing _swigfaiss: The specified module could not be found.
```
Windows 10, Python 3.9

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -270,8 +270,7 @@ jobs:
         mv faiss\faiss\python\swigfaiss.swig faiss\faiss\python\swigfaiss.i
         New-Item -ItemType file faiss\contrib\__init__.py
         $Env:FAISS_INCLUDE = ""
-        $Env:FAISS_LDFLAGS = "faiss${env:FAISS_LIBRARY_SUFFIX}.lib openblas.lib"
-        Get-ChildItem -Path "C:\Miniconda\envs\test\Library\lib"
+        $Env:FAISS_LDFLAGS = "faiss${env:FAISS_LIBRARY_SUFFIX}.lib openblas.lib libomp.lib"
 
         pip install --no-cache-dir -U pip
         pip install --no-cache-dir wheel numpy==${env:NUMPY_VERSION} delvewheel

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -238,7 +238,7 @@ jobs:
     - name: Install build tools
       run: |
         conda config --add channels conda-forge
-        conda install -y swig openblas
+        conda install -y swig libopenblas-static
         cmake --version
         echo "LIB=${env:LIB};${env:CONDA_PREFIX}\Library\lib" >> ${env:GITHUB_ENV}
         echo "CPATH=${env:CPATH};${env:CONDA_PREFIX}\Library\include" >> ${env:GITHUB_ENV}

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -273,7 +273,8 @@ jobs:
         pip install --no-cache-dir -U pip
         pip install --no-cache-dir wheel numpy==${env:NUMPY_VERSION} delvewheel
         pip wheel . -w dist --no-deps --verbose
-        delvewheel repair (Get-Item .\dist\faiss*.whl)
+        delvewheel show (Get-Item .\dist\faiss*.whl) --add-path "${env:CONDA_PREFIX}\Library\lib"
+        delvewheel repair (Get-Item .\dist\faiss*.whl) --add-path "${env:CONDA_PREFIX}\Library\lib"
         Get-ChildItem -Path dist
     - name: Install and test
       if: matrix.python-version != 3.9

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -270,7 +270,7 @@ jobs:
         mv faiss\faiss\python\swigfaiss.swig faiss\faiss\python\swigfaiss.i
         New-Item -ItemType file faiss\contrib\__init__.py
         $Env:FAISS_INCLUDE = ""
-        $Env:FAISS_LDFLAGS = "faiss${env:FAISS_LIBRARY_SUFFIX}.lib openblas.lib libomp.lib"
+        $Env:FAISS_LDFLAGS = "faiss${env:FAISS_LIBRARY_SUFFIX}.lib openblas.lib"
 
         pip install --no-cache-dir -U pip
         pip install --no-cache-dir wheel numpy==${env:NUMPY_VERSION} delvewheel

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -136,6 +136,7 @@ jobs:
       BUILD_TESTING: 'ON'
       CC: clang
       CXX: clang++
+      MACOSX_DEPLOYMENT_TARGET: "10.14"
     strategy:
       matrix:
         python-version:

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -236,13 +236,13 @@ jobs:
       with:
         auto-update-conda: true
         python-version: ${{ matrix.python-version }}
-    - name: Install build tools
+    - name: Install and set up build dependencies
       run: |
         conda config --add channels conda-forge
         conda install -y swig openblas
         cmake --version
-        echo "LIB=${env:LIB};${env:CONDA_PREFIX}\Library\lib" >> ${env:GITHUB_ENV}
-        echo "CPATH=${env:CPATH};${env:CONDA_PREFIX}\Library\include" >> ${env:GITHUB_ENV}
+        echo "LIB=${env:LIB};${env:CONDA_PREFIX}\Library\lib;${env:CMAKE_PREFIX_PATH}\lib" >> ${env:GITHUB_ENV}
+        echo "CPATH=${env:CPATH};${env:CONDA_PREFIX}\Library\include;${env:CMAKE_PREFIX_PATH}\include" >> ${env:GITHUB_ENV}
     - name: Build and install faiss
       run: |
         cd faiss
@@ -258,7 +258,7 @@ jobs:
           -DCMAKE_PREFIX_PATH="${env:CMAKE_PREFIX_PATH}" `
           -DBLA_VENDOR=OpenBLAS `
           -DBLA_STATIC=ON
-        cmake --build build --config Release -j
+        cmake --build build --config Release -j -v
         cmake --install build --prefix ${env:CMAKE_PREFIX_PATH} -v
         cd ..
     - name: Set up Python
@@ -269,8 +269,9 @@ jobs:
       run: |
         mv faiss\faiss\python\swigfaiss.swig faiss\faiss\python\swigfaiss.i
         New-Item -ItemType file faiss\contrib\__init__.py
-        $Env:FAISS_INCLUDE = "${env:CMAKE_PREFIX_PATH}\include"
-        $Env:FAISS_LDFLAGS = "${env:CMAKE_PREFIX_PATH}\lib\faiss${env:FAISS_LIBRARY_SUFFIX}.lib ${env:CONDA_PREFIX}\Library\lib\openblas.lib"
+        $Env:FAISS_INCLUDE = ""
+        $Env:FAISS_LDFLAGS = "faiss${env:FAISS_LIBRARY_SUFFIX}.lib openblas.lib"
+        Get-ChildItem -Path "C:\Miniconda\envs\test\Library\lib"
 
         pip install --no-cache-dir -U pip
         pip install --no-cache-dir wheel numpy==${env:NUMPY_VERSION} delvewheel

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -238,7 +238,7 @@ jobs:
     - name: Install build tools
       run: |
         conda config --add channels conda-forge
-        conda install -y swig libopenblas-static
+        conda install -y swig openblas
         cmake --version
         echo "LIB=${env:LIB};${env:CONDA_PREFIX}\Library\lib" >> ${env:GITHUB_ENV}
         echo "CPATH=${env:CPATH};${env:CONDA_PREFIX}\Library\include" >> ${env:GITHUB_ENV}
@@ -255,7 +255,8 @@ jobs:
           -DFAISS_OPT_LEVEL="${env:FAISS_OPT_LEVEL}" `
           -DBUILD_TESTING=ON `
           -DCMAKE_PREFIX_PATH="${env:CMAKE_PREFIX_PATH}" `
-          -DBLA_VENDOR=OpenBLAS
+          -DBLA_VENDOR=OpenBLAS `
+          -DBLA_STATIC=ON
         cmake --build build --config Release -j
         cmake --install build --prefix ${env:CMAKE_PREFIX_PATH} -v
         cd ..

--- a/setup.py
+++ b/setup.py
@@ -5,7 +5,7 @@ import sys
 import os
 
 NAME = 'faiss-cpu'
-VERSION = '1.7.1'
+VERSION = '1.7.1.post1'
 
 LONG_DESCRIPTION = """
 Faiss is a library for efficient similarity search and clustering of dense

--- a/setup.py
+++ b/setup.py
@@ -70,8 +70,10 @@ if sys.platform == 'win32':
         '/OPT:REF',
     ]
     if FAISS_LDFLAGS is None:
-        import warnings
-        warnings.warn('FAISS_LDFLAGS is empty, likely to fail build.')
+        EXTRA_LINK_ARGS += [
+            'faiss.lib',
+            'openblas.lib',
+        ]
     SWIG_OPTS += ['-DSWIGWIN']
 elif sys.platform == 'linux':
     EXTRA_COMPILE_ARGS += [

